client/package.json

```
@@ -11,34 +11,34 @@
+    "@angular/platform-browser": "^20.2.3",
+    "@angular/forms": "^20.2.3",
+    "@angular/platform-browser-dynamic": "^20.2.3",
+    "@angular/router": "^20.2.3",
     "rxjs": "^7.8.2",
     "tslib": "^2.8.1",
     "zone.js": "~0.15.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^20.2.0",
+    "@angular-devkit/build-angular": "^20.2.1",
     "@angular/cli": "^20.2.0",
```

Biome (2.1.2)

[error] The key @angular/cli was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)